src/services/LegacyDomainService.ts

```diff
@@ -0,0 +1,177 @@
+import { Context, HttpRequest } from "@azure/functions";
+import {
+  Address,
+  DomainFindOptions,
+  DomainId,
+  Maybe,
+} from "@zero-tech/data-store-core";
+import { MongoDbService } from "@zero-tech/data-store-core/lib/database/mongo/mongoDbService";
```

Comment:
You shouldn't have to go deep into this imported repo to get the type you need. Make sure `data-store-core` exports `MongoDbService` at the top level

Reply:
I cannot modify data-store-core its a separate package

Reply:
@JamesEarle it may not have been exposed at the top level by `data-store-core`